pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/channel/BaseChannel.kt

```diff
@@ -754,6 +754,10 @@ abstract class BaseChannel<C : Channel, M : Message>(
         typingSent = value
     }
 
+    internal fun getTypingSent(): Instant? {
```

Comment:
nit: you can just change `val typingTest` to internal

Reply:
Makes sense. I will do this and remove getTypingSend and setTypingSend methods.